components/NewFooter/index.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import Footer from './index';
+import type { Meta as MetaObj } from '@storybook/react';
+
+type Meta = MetaObj<typeof Footer>;
+
+export const Default = () => <Footer />;
+
+export default {
+  title: 'New Layout/Footer',
```

Comment:
We shouldn't add `title`.

Reply:
OK why.

Reply:
Because that's what is on the contributing guidelines / no other component is doing that.

Reply:
Thanks for the explaining. But I missed to write "not".